puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/duffy.groovy

```diff
@@ -1,21 +1,41 @@
+import groovy.json.JsonSlurper
+
 def provision() {
     dir('foreman-infra') {
-        git url: 'https://github.com/ehelms/foreman-infra.git', branch: 'ci-centos-org'
+        git url: 'https://github.com/theforeman/foreman-infra.git'
     }
 
-	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
+  	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
         runPlaybook('provision.yml', 'localhost')
     }
 }
 
 def deprovision() {
-    if(fileExists('foreman-infra')) {
-        dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
+    if (fileExists('foreman-infra')) {
+      	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
             runPlaybook('deprovision.yml', 'localhost')
-        }
+        }f
     }
 }
 
 def cico_inventory(relative_dir = '') {
-	return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/cico_inventory'
+  	return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/cico_inventory'
+}
+
+def ssh_config(relative_dir = '') {
+    return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/ssh_config'
+}
+
+def wrap_shell(command = '') {
```

Comment:
Thgouths on color_shell ?

Reply:
I'm fine with that. I just used this because of the wrap class being used. I think color is more descriptive as to the outcome.